project/PythonBuildPlugin.scala

```diff
@@ -57,19 +57,29 @@ object PythonBuildPlugin extends AutoPlugin {
     copySources(Test / pythonSource, Python / test / target, false)
   )
 
-  val buildWhl = Def.task {
-    val buildDir = (Python / target).value
-    val retcode = pySetup.toTask(" build bdist_wheel").value
-    if(retcode != 0) throw new RuntimeException(s"'python setup.py' returned $retcode")
-    val whls = (buildDir / "dist" ** "pyrasterframes*.whl").get()
-    require(whls.length == 1, "Running setup.py should have produced a single .whl file. Try running `clean` first.")
-    whls.head
-  }
+  val buildWhl = Def.sequential(
+    (Compile / assembly), // Needed because SBT will try to parallelize this step out of sequence.
+    Def.task {
+      val log = streams.value.log
+      val buildDir = (Python / target).value
+      val asmbl = (Compile / assembly).value
+      val dest = buildDir / "deps" / "jars" / asmbl.getName
+      IO.copyFile(asmbl, dest)
```

Comment:
@vpipkt The file is getting copied to what I think is the right place, and would expect the `MANIFEST.in` to find it, but it's not getting wrapped up into the `.whl` file. 🤔 Help?

Reply:
@vpipkt Now not convinced this isn't an sbt "over parallelizing things" issue... hold off looking at this.